Animal.php

```diff
@@ -0,0 +1,323 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="index.css">
+    <link rel="stylesheet" href="Gallery.css">
+    <link rel="stylesheet" href="Animal.css">
+    <link rel="stylesheet" href="animal2.css">
+
+</head>
+<body>
+      <!-- ########################## MAIN MANU ################################### -->
+      <?php include 'header.php';?>
+<!-- ########################## Gallery ################################### -->
+<div class="gallery_h1">
+    <h1> Birds</h1>
+</div>
+<br><br>
+  
+<div class="responsive">
+    <div class="gallery">
+      <a href="img_5terre.jpg">
```

Comment:
- [ ] Please remove the image file and create a file that will display details of a single animal and add it to this link
You can name it  animal_details.php